src/config.ts

```diff
@@ -10,9 +10,9 @@ declare global {
 }
 
 const config: Env = {
-  API_URL: window.ENV.API_URL || process.env.REACT_APP_API_URL,
```

Comment:
this issue has been fixed, it should work correctly on localhost with "window.ENV.*" presented now, revert it please